tests/KubernetesClient.Tests/KubernetesClient.Tests.csproj

```diff
@@ -1,11 +1,15 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <IsPackable>false</IsPackable>
+    <LangVersion>8</LangVersion>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>..\..\src\KubernetesClient\kubernetes-client.snk</AssemblyOriginatorKeyFile>
```

Comment:
why do we need strong name here?

Reply:
In order to test `internal` methods from test project, you need to mark test project friend of main one. Since main one is strongly signed, test assembly also needs to be strongly signed.